dt-backend/test/model/user/user.e2e-spec.ts

```diff
@@ -16,32 +14,61 @@ describe('UserController e2e', () => {
     await app.init();
   });
 
-  it('/GET ', () => {
-    return request(app.getHttpServer()).get('/user').expect(200).expect('[]');
-  });
-
-  it('/user/signUp ', () => {
+  test('/update/:id', async () => {
+    // create user
     const user = {
       username: 'john',
       password: 'john',
       mail: 'john@john.com',
     };
-    return request(app.getHttpServer())
+    //send create request for john
+    const res = await request(app.getHttpServer())
       .post('/user/signUp')
       .send(user);
+
+    //update user john to joe
+    const updateUser = {
+      username: 'joe',
+      password: 'joe',
+      mail: 'joe@joe.com',
+    };
+    const update = await request(app.getHttpServer())
+      .patch('/user/update/' + res.body.id)
+      .send(updateUser);
+
+    //delete user
+    await request(app.getHttpServer())
+      .delete('/user/delete/' + res.body.id)
+      .send(res.body.id);
   });
 
-  it('/update', () => {
+  test('delete/:id', async () => {
+    const user = {
+      username: 'john',
+      password: 'john',
+      mail: 'john@john.com',
+    };
+    const res = await request(app.getHttpServer())
+      .post('/user/signUp')
+      .send(user)
+      .expect(200);
     return request(app.getHttpServer())
-      .get('/user/update')
-      .expect(200)
-      .expect('');
+      .delete('/user/delete/' + res.body.id)
+      .send(res.body.id)
+      .expect(200);
   });
 
-  it('delete', () => {
-    return request(app.getHttpServer())
-      .get('/user/delete')
-      .expect(200)
-      .expect('');
+  test('/user/:id/audit', async () => {
+    //create user to add audit to
+    const user = {
+      username: 'john',
+      password: 'john',
+      mail: 'john@john.com',
+    };
+    const res = await request(app.getHttpServer())
+      .post('/user/signUp')
+      .send(user)
+      .expect(200);
+    res.body.audit.append()
```

Comment:
This is yet incomplete. There is stuff to be done here.
You don't do anything with the audit here yet 🤷. You just create a user (and don't forget to delete the user at the bottom - as always [along with the Audit, as well]!